### PR TITLE
Add model timeout

### PR DESCRIPTION
**Pull request by pavlejacovicHTEC**

### CODE REVIEW

File: .github/workflows/performance.yaml [hunk 1]
```diff
@@ -46,6 +50,7 @@ jobs:
       performance_reports_repo: ${{ github.event.inputs.performance_reports_repo || 'ROCmSoftwarePlatform/migraphx-reports' }}
       benchmark_utils_repo: ${{ github.event.inputs.benchmark_utils_repo || 'ROCmSoftwarePlatform/migraphx-benchmark-utils' }}
       organization: ${{ github.event.inputs.organization || 'AMD' }}
+      model_timeout: ${{ github.event.inputs.model_timeout || '30m' }}
```
Comment (umangyadav) [inline .github/workflows/performance.yaml:53]: Did we update `performance_tests.sh` script to take in timeout parameter ?

Comment (pavlejacovicHTEC) [reply .github/workflows/performance.yaml:53]: Yes. That change is currently under this PR: [](https://github.com/ROCmSoftwarePlatform/migraphx-benchmark-utils/pull/14/commits/e5094dedf6dae2317e2d12a2c103251f5d1c28a1#diff-f797eb33b364b5906e3ce3023348ef37dd7db019c528021d9112dfd991ec70bf)


The line I am reffering to is: `timeout $1 $DRIVER perf $extra $TARGETOPT $SAVED_MODELS/$savefile > ${tag}.out 2> ${tag}.err`

Comment (pavlejacovicHTEC) [reply .github/workflows/performance.yaml:53]: You have to merge all three of these PR's in order for model_timeout to work:
1. https://github.com/ROCmSoftwarePlatform/migraphx-benchmark/pull/13
2. https://github.com/ROCmSoftwarePlatform/AMDMIGraphX/pull/1649
3. https://github.com/ROCmSoftwarePlatform/migraphx-benchmark-utils/pull/14

This last one is fix related to something else, but I included the timeout in performance_tests.sh script inside

**Review by causten — APPROVED** [review]

File: .github/workflows/performance.yaml (hunk 1 above)
Comment (umangyadav) [reply .github/workflows/performance.yaml:53]: Okay. I've put some comments in other https://github.com/ROCmSoftwarePlatform/migraphx-benchmark-utils/pull/14

**Review by TedThemistokleous — APPROVED** [review]

**Pull request merged**